tools/llm_bench/llm_bench_utils/model_utils.py

```diff
@@ -92,6 +98,43 @@ def get_image_param_from_prompt_file(args):
     return image_param_list
 
 
+def get_audio_param_from_media_file(args):
+    audio_param_list = []
+    if args['media'] is None and args['prompt_file'] is None:
+        raise RuntimeError('== media and prompt file is empty ==')
+    elif args['media'] is not None and args['prompt_file'] is not None:
+        raise RuntimeError('== media and prompt file should not exist together ==')
+    else:
+        if args['media'] is not None:
+            if args['media'] != '':
+                audio_param_list.append({'media' : args['media']})
+            else:
+                raise RuntimeError('== media path should not be empty string ==')
+        else:
+            input_media_list = args['prompt_file']
+            for input_media in input_media_list:
+                if input_media.endswith('.jsonl'):
```

Comment:
I believe all previous prompt files are also jsonl, right? is it possible to organize some utility for reading this file format for avoid code duplication for different tasks?